src/components/Calculator.scss

```diff
@@ -0,0 +1,42 @@
+.App {
+  display: flex;
+  justify-content: center;
+  margin-top: 25%;
+  width: 100vw;
+}
+
+.calculator {
+  display: flex;
+  flex-direction: column;
+  max-width: 25%;
+}

```

Comment:
- [ ] Your site slightly overflows. Kindly fix this 🙏 

![20221103-221945_capture](https://user-images.githubusercontent.com/74814780/199837398-560d3f5a-6e6a-47ef-bd4a-7a6557812827.gif)